.size-snapshot.json

```diff
@@ -1,21 +1,26 @@
 {
   "dist/react-beautiful-dnd.js": {
-    "bundled": 391074,
```

Comment:
Could you please let me know:

- what the change in numbers is between the current dev and this branch? Ie how many kbs do we loose moving to babel 7
- could you also compare this again the current master? I will put these figures in the release notes

Reply:
❤️!

Reply:
You can compare the last commit against the first commit in this PR. The first one is only snapshot update.

```diff
{
  "dist/react-beautiful-dnd.js": {
-    "bundled": 380027,
-    "minified": 143820,
-    "gzipped": 40855
+    "bundled": 356054,
+    "minified": 134070,
+    "gzipped": 37695
  },
  "dist/react-beautiful-dnd.min.js": {
-    "bundled": 343989,
-    "minified": 130858,
-    "gzipped": 36610
+    "bundled": 320016,
+    "minified": 121108,
+    "gzipped": 33429
  },
  "dist/react-beautiful-dnd.esm.js": {
-    "bundled": 180665,
-    "minified": 91821,
-    "gzipped": 23548,
+    "bundled": 179908,
+    "minified": 90873,
+    "gzipped": 23472,
    "treeshaked": {
      "rollup": {
-        "code": 76262,
-        "import_statements": 874
+        "code": 75679,
+        "import_statements": 733
      },
      "webpack": {
-        "code": 78237
+        "code": 77480
      }
    }
  }
}
```

Reply:
Here's against master
```diff
{
  "dist/react-beautiful-dnd.js": {
-    "bundled": 400269,
-    "minified": 151652,
-    "gzipped": 43139
+    "bundled": 356054,
+    "minified": 134070,
+    "gzipped": 37695
  },
  "dist/react-beautiful-dnd.min.js": {
-    "bundled": 358556,
-    "minified": 134751,
-    "gzipped": 38201
+    "bundled": 320016,
+    "minified": 121108,
+    "gzipped": 33429
  },
  "dist/react-beautiful-dnd.esm.js": {
-    "bundled": 177759,
-    "minified": 89807,
-    "gzipped": 22649,
+    "bundled": 179908,
+    "minified": 90873,
+    "gzipped": 23472,
    "treeshaked": {
-      "rollup": 78288,
+      "rollup": {
+        "code": 75679,
+        "import_statements": 733
+      },
-      "webpack": 80060
+      "webpack": {
+        "code": 77480
+      }
    }
  }
}
```

Reply:
`"code": 75679,`

Reply:
just so i understand - if a user has no shared deps with the library then they take the 179kb bundle size. If they already have all the deps they get 75kb. Is that correct?

Reply:
Not quite. Bundled, minified, gzipped are sizes of what we provide in rollup config + terser + gzip.

So in production the library size with dependencies will be 121KB to parse and 33KB to load (minified umd).

If user already have all dependencies then the size will be 90KB to parse and 23KB to load subtract development stuff like `process.env.NODE_ENV` (size snapshot provides production replacement only for treeshaked size)

Treeshaked size doesn't make a lot of sense for this library because as you said there is nothing unused.

Reply:
I think it would be interesting to know what the lowest possible size gzip could be. Ie: if with process.env.NODE_ENV === production and with all dependencies de duplicated.

Reply:
Don't need to do this here though!

Reply:
I am still a confused by this:

"treeshaked": {
-      "rollup": 78288,
+      "rollup": {
+        "code": 75679,
+        "import_statements": 733
+      },
-      "webpack": 80060
+      "webpack": {
+        "code": 77480
+      }
    }